### ISSUE TITLE
spanner: Update backups samples once metadata decoding support is implemented

### ISSUE THREAD

**Issue by skuruppu** [GoogleCloudPlatform/php-docs-samples#1050]
In the backup and restore code samples (#1046), we wanted to show users how to decode the metadata protos that contain information about a backup creation, restore backup and optimize database operations. There's currently no mechanism to do that in the PHP library. [#2536](https://github.com/googleapis/google-cloud-php/pull/2536) is an attempt at doing this.

The two samples to fix are:

1. `list_backup_operations.php`
```
print("Backup $metadataName on database $metadataDatabase is $progressPercent% pending" . PHP_EOL);
```

2. `list_database_operations.php`
```
print("Database $metadataName restored from backup is $progressPercent% optimized." . PHP_EOL);
```

**Comment by skuruppu** [comment]
I think the samples are now fixed. If you think all the work is done, please close this issue @ava12.

**Comment by ava12** [comment]
I cannot close it (i don't see the "close" button). I guess @AVaksman can.

**Comment by AVaksman** [comment]
This issue is fixed with #1088
